scipy/special/_basic.py

```diff
@@ -13,7 +13,7 @@
 from . import _ufuncs
 from ._ufuncs import (mathieu_a, mathieu_b, iv, jv, gamma,
                       psi, hankel1, hankel2, yv, kv, poch, binom,
-                      _stirling2_inexact)
+                      _stirling2_inexact, _lpn, _lpmn, _clpmn, _lqn, _lqmn)
```

Comment:
It would be really cool if we could have

```python
from ._gufuncs import _lpn, _lpmn, _clpmn, _lqn, _lqmn
```

it makes it much more clear at a glance that these are different from the ordinary ufuncs. There's a few ways this could be done.

Reply:
Yeah, that's fine.